repo/packages/M/minio/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-medium": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-large": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png"
```

Comment:
Also, to make sure the service icons render properly in the UI, use the specs from https://github.com/mesosphere/universe#images